### PR TITLE
fix/highlighted markdown

### PR DESCRIPTION
**Pull request by sywhb**
- fix: `&npsp;` breaks highlight
- fix: wrap link with `==`
- fix: sentry report on unable to find highlight

### CODE REVIEW

File: packages/api/src/utils/highlightGenerator.ts [hunk 1]
```diff
@@ -419,7 +422,7 @@ export function makeHighlightNodeAttributes(
           isParagraphStart && !i && quote && (quote += '\n')
           quote += text
         }
-
+        console.log('quote', quote)
```
Comment (jacksonh) [inline packages/api/src/utils/highlightGenerator.ts:425]: not sure if you need this anymore

Comment (sywhb) [reply packages/api/src/utils/highlightGenerator.ts]: haha, i'm removing this and then i saw your comment @jacksonh

**Review by jacksonh — APPROVED** [review]
I have a feeling we need to start building up test cases for this as there are going to be a lot of edge cases.

**Comment by jacksonh** [comment]
LGTM
